pkgs/top-level/perl-packages.nix

```diff
@@ -353,7 +353,7 @@ let
       url = "mirror://cpan/authors/id/M/ML/MLEHMANN/AnyEvent-AIO-1.1.tar.gz";
       sha256 = "0svh0mlp17g0ypq8bgs3h3axg8v7h0z45hryacgn6q8mcj65n43b";
     };
-    propagatedBuildInputs = [ AnyEvent IOAIO ];
```

Comment:
this may be wrong, it is one of the packages you recently added, and cpan2nix has no glue. it took deps from cpan metadata, which is often invalid